wazimap_ng/datasets/views.py

```diff
@@ -97,7 +97,7 @@ def dataset_upload(request, dataset_id):
     update_indicators = request.POST.get('update', False)
     upload_task = async_task(
         "wazimap_ng.datasets.tasks.process_uploaded_file",
-        datasetfile_obj, dataset,
+        datasetfile_obj, dataset, reset=True,
```

Comment:
can we add another flag in the data. `overwrite` ? and just do it when the flag is set to true.